apps/src/templates/watchers/Watchers.jsx

```diff
@@ -1,3 +1,4 @@
+/* global appOptions */
```

Comment:
this global shouldn't be necessary. appOptions.app should be passed in via `setPageConstants`, the same way `showDebugWatch` is passed in.